docs/community/com-config/com-config-fb-scaling.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Scaling FeatureBase
+layout: default
+parent: Community configuration
+grand_parent: Community
+---
+
+## How do I scale FeatureBase Community?
+
+{: .note}
+You can also [Resize a FeatureBase Cluster](/docs/community/com-config/old-resize-cluster)
+
+Generally, scaling is handled by starting or stopping Ingester instances.
+
+## Scale FeatureBase up
```

Comment:
Where is this information coming from? I see scaling is not supported here: https://docs.featurebase.com/docs/community/com-config/old-scaling/

Reply:
Yep that'll be in "part 2" of this revision. As I say, things were getting complex so it felt a good time to nail down what was there and then I'll sort the rest out in another PR :)